packages/net/http/_client_connection.pony

```diff
@@ -181,6 +181,14 @@ actor _ClientConnection is HTTPSession
     """
     None
 
+  be dispose() =>
+    """
+    Close the connection from the client end.
+    """
+    match _conn
+      | let c: TCPConnection => c.dispose()
```

Comment:
i think once this is update, this is good to merge. just drop me a note when its done and i'll merge @pdtwonotes.